selfdrive/car/volkswagen/pqcan.py

```diff
@@ -91,7 +91,7 @@ def create_acc_accel_control(packer, bus, acc_type, acc_enabled, accel, acc_cont
   return commands
 
 
-def create_acc_hud_control(packer, bus, acc_hud_status, set_speed, lead_distance):
+def create_acc_hud_control(packer, bus, acc_hud_status, set_speed, lead_distance, distance):
   values = {
     "ACA_StaACC": acc_hud_status,
     "ACA_Zeitluecke": 2,
```

Comment:
@sunnyhaibin Per our DM conversation, this should be the time gap signal for PQ and can get a similar change. I feel like I probably would have set the PQ displayed default at 3 of 5 bars just like MQB, so the PQ signal might be zero instead of one based. We'll need to ask @dkiiv or another of the PQ crew to validate. There's some focus-on-change logic we should also implement someday but it's not necessary for a first pass.